src/components/List/List.stories.tsx

```diff
@@ -109,7 +146,7 @@ export const cardList = () => (
       type={select("List Type", ListTypes, ListTypes.Unordered)}
       modifiers={boolean("List Styling", false) ? null : ["no-list-styling"]}
     >
-      {cards}
+      <p>test</p> {cards}
```

Comment:
This renders fine without the test `p` tag.

Reply:
Oops; I should have deleted that before pushing the code..  Fixed it now.